spec/models/like_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe '#update_post_likes_counter' do
+    let(:user) { User.create(name: 'John Doe', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'teacher') }
+    let(:post) { Post.create(title: 'Test Post', text: 'This is a test post', author_id: user.id) }
+
+    it 'increments post likes counter after creating a like' do
+      expect { Like.create(user_id: user.id, post_id: post.id) }.to change { post.reload.likes_counter }.by(1)
+    end
+
+    it 'decrements post likes counter after destroying a like' do
+      like = Like.create(user_id: user.id, post_id: post.id)
+      expect { like.destroy }.to change { post.reload.likes_counter }.by(-1)
+    end
```

Comment:
- [ ] Hi @enocol, I'm also getting the same errors as mentioned by the previous reviewer (see image). Some of your tests are failing because the **likes** table is not created when runed the **`migration`**. I notice that the migrations files that help recreate the tables in your database are deleted from your branch and the **likes** table is not present in your `schema.rb` file, Kindly add them back.
![Screenshot from 2024-01-16 15-08-47](https://github.com/enocol/Blogger/assets/91142432/334ffc67-b870-4c34-83d1-2dcec86360a6)